pulpcore/pulpcore/exceptions/repository.py

```diff
@@ -0,0 +1,45 @@
+from gettext import gettext as _
+
+from .base import PulpException
+
+
+class RepostoryVersionExistsError(PulpException):
```

Comment:
Let's consider renaming as: `ResourceImmutableError`.  I think it more accurately and generically represents the error condition.